lib/algorithm.ts

```diff
@@ -353,3 +354,45 @@ function hazing({ tweet, sentiment }: TweetData): Rank {
     score: 0,
   }
 }
+
+/**
+ * U.S. politics is so awesome, and everybody should love to read about it,
+ * no matter their background, interests, or country of residence.
+ * You WILL form an opinion on us!
+ */
+function usPolitics({ tweet, sentiment }: TweetData): Rank {
+  // List of hot topics, power posters, and coronavirus conspiracy
+  const topics = [
+    "🇺🇸", "american?", "patriot", "president", "states", "psyop",
+    "conservatives?", "democrat", "demonrat", "republican", "crt", "woke\\w*",
+    "wokeness", "abortion", "pro-life", "authoritarian", "drag", "ideology",
+    "trump", "biden", "obama", "aoc", "cruz", "bernie", "sanders", "mtg",
```

Comment:
both aoc and mtg are terms often used by people in the nerd culture. They should be bullied instead of boosted

Reply:
Showing these tweets to more people means more surface area for the authors to be bullied. LGTM